.circleci/tests/test_smoothing.py

```diff
@@ -73,58 +77,68 @@ def test_smoothing_Nifti(data_dir):
-# def test_smoothing_Cifti(data_dir, sigma_lx):
-#     # Specify inputs
-#      data_dir = os.path.join(data_dir,
-#                           "fmriprepwithfreesurfer")
-#     in_file = data_dir + "/fmriprep/sub-colornest001/ses-1/func/" \
-#         "sub-colornest001_ses-1_task-rest_run-1_space-fsLR_den-91k_bold.dtseries.nii"
-
-#     # Let's get into a temp dir
-#     tmpdir = tempfile.mkdtemp()
-#     os.chdir(tmpdir)
-
-#     # Run AFNI'S FWHMx
-#     fwhm = afni.FWHMx()
-#     fwhm.inputs.in_file = in_file
-#     fwhm.inputs.detrend = True
-#     results = fwhm.run()
-#     in_file_smoothness = results.outputs.fwhm
-
-#     # Smooth the data
-#     smooth_data = pe.Node(CiftiSmooth(  # Call connectome workbench to smooth for each
-#         #  hemisphere
-#         sigma_surf=sigma_lx,  # the size of the surface kernel
-#         sigma_vol=sigma_lx,  # the volume of the surface kernel
-#         direction='COLUMN',  # which direction to smooth along@
-#         right_surf=pkgrf(  # pull out atlases for each hemisphere
-#             'xcp_d', 'data/ciftiatlas/'
-#             'Q1-Q6_RelatedParcellation210.R.midthickness_32k_fs_LR.surf.gii'
-#         ),
-#         left_surf=pkgrf(
-#             'xcp_d', 'data/ciftiatlas/'
-#             'Q1-Q6_RelatedParcellation210.L.midthickness_32k_fs_LR.surf.gii'
-#         )),
-#         name="cifti_smoothing")
-#     smooth_data.inputs.in_file = in_file
-#     smooth_data.inputs.out_file = 'test.dtseries.nii'
-#     results = smooth_data.run()
-#     out_file = results.outputs.out_file
-
-#     # Run AFNI's FWHMx on the smoothed data
-#     fwhm = afni.FWHMx()
-#     fwhm.inputs.in_file = out_file
-#     fwhm.inputs.detrend = True
-#     results = fwhm.run()
-#     out_file_smoothness = results.outputs.fwhm
-
-#     print(in_file_smoothness)
-#     print(out_file_smoothness)
-
-#     return
+def test_smoothing_Cifti(data_dir, sigma_lx=fwhm2sigma(6)):
+
+    # Specify inputs
+    right_surf = pkgrf(  # pull out atlases for each hemisphere
+        'xcp_d', 'data/ciftiatlas/'
+        'Q1-Q6_RelatedParcellation210.R.midthickness_32k_fs_LR.surf.gii'
+    )
+    left_surf = pkgrf(
+        'xcp_d', 'data/ciftiatlas/'
+        'Q1-Q6_RelatedParcellation210.L.midthickness_32k_fs_LR.surf.gii'
+    )
+
+    data_dir = os.path.join(data_dir,
+                            "fmriprepwithfreesurfer")
+    in_file = os.path.join(
+        data_dir,
+        "fmriprep/sub-colornest001/ses-1/func",
+        "sub-colornest001_ses-1_task-rest_run-1_space-fsLR_den-91k_bold.dtseries.nii")
+    # What's the smoothness?
+    in_file_smoothness = os.popen("wb_command -cifti-estimate-fwhm " + in_file
+                                  + " -surface CORTEX_LEFT " + left_surf
+                                  + " -surface CORTEX_RIGHT " + right_surf
+                                  + " -whole-file -merged-volume").read()
+
+    # Let's get into a temp dir
+    tmpdir = tempfile.mkdtemp()
+    os.chdir(tmpdir)
```

Comment:
Rather than changing the directory, can you prepend tmpdir to the output filename and set the node's based to tmpdir?